dc-model/src/main/java/de/digitalcollections/model/api/identifiable/entity/agent/Person.java

```diff
@@ -1,9 +1,12 @@
 package de.digitalcollections.model.api.identifiable.entity.agent;
 
+import de.digitalcollections.model.api.identifiable.agent.FamilyName;
+import de.digitalcollections.model.api.identifiable.agent.GivenName;
 import de.digitalcollections.model.api.identifiable.entity.Entity;
 import de.digitalcollections.model.api.identifiable.entity.agent.enums.Gender;
 import de.digitalcollections.model.api.identifiable.entity.geo.GeoLocation;
 import java.time.LocalDate;
+import java.util.List;
 import org.wikidata.wdtk.datamodel.interfaces.TimeValue;
 
 /** Human being that has certain capacities or attributes constituting personhood. */
```

Comment:
Braucht es die Vererbung mit `Entity` wirklich? Kann man sich wohl sparen.

Reply:
nein. entfernt (auch in corporatebody)